### PR TITLE
Update Benchmark documentation [skip ci]

### PR DESCRIPTION
**Pull request by davydovanton**

### CODE REVIEW

File: lib/benchmark.rb [hunk 1]
```diff
@@ -219,6 +219,9 @@ def bm(label_width = 0, *labels, &blk) # :yield: report
   # Because #bmbm takes two passes through the tests, it can
   # calculate the required label width.
   #
+  # Also #bmdm take +width+ argument. This argument is a initial value
+  # for the label offset used in formatting;
```
Comment (eregon) [inline lib/benchmark.rb]: This argument is legacy and it's better to not provide it as the method can compute it itself.
For `#bm`, it it useful though.

Comment (davydovanton) [inline lib/benchmark.rb]: no problem. Fixed

File: lib/benchmark.rb [hunk 2]
```diff
@@ -281,7 +281,20 @@ def bmbm(width = 0) # :yield: job
 
   #
   # Returns the time used to execute the given block as a
-  # Benchmark::Tms object.
+  # Benchmark::Tms object. Takes +label+ opliton.
```
Comment (hsbt) [inline lib/benchmark.rb]: 's/opliton/option/' ?

**Comment by davydovanton** [comment]
Oh, sorry you're right

**Comment by davydovanton** [comment]
@hsbt you are forgot add reference to me in changelog :'(

**Pull request closed without merging**